pydantic_ai_slim/pydantic_ai/ui/vercel_ai/response_types.py

```diff
@@ -145,6 +148,21 @@ class ToolOutputErrorChunk(BaseChunk):
     dynamic: bool | None = None
 
 
+class ToolApprovalRequestChunk(BaseChunk):
```

Comment:
Do you know if it's feasible for us to yield this automatically when https://ai.pydantic.dev/deferred-tools/#human-in-the-loop-tool-approval is used? I haven't looked into how approval requests work with Vercel AI, yet.